play/type-racer/app.js

```diff
@@ -120,11 +188,17 @@ function stopLightStart() {
 /**
  * Start the game.
  */
-function startGame() {
+async function startGame() {
     // prevent startGame() from being called multiple times before ending
     if (gameStarted) {return;}
     gameStarted = true;
 
+    // wait to finish reading in the text file
+    let promise = await readTextFile(WORD_LIST);
+
+    // let the user know if an error occurred
+    if (promise == -1) {alert("Error: Input text file could not be read");}
```

Comment:
Should the user be alerted of network errors? It should either be a user friendly error message, or use ```console.error()``` to log errors.